test/events/EventTileFactory-test.ts

```diff
@@ -85,12 +135,20 @@ describe("pickFactory", () => {
             expect(pickFactory(voiceBroadcastChunkEvent, client, false)).toBeUndefined();
         });
 
-        it("should return a Function for a voice broadcast stopped event", () => {
-            expect(pickFactory(voiceBroadcastStoppedEvent, client, true)).toBeInstanceOf(Function);
+        it("should return a TextualEventFactory for a voice broadcast stopped event", () => {
+            expect(pickFactory(voiceBroadcastStoppedEvent, client, true)).toBe(TextualEventFactory);
+        });
```

Comment:
To me this test looks identical to the test in line 120, is this definitely how it should be? Just wondering if the third argument is meant to be false or, if that's not the case, whether it might be worth "lifting" it up to the level of the describe block containing it to indicate this is the same in both cases, when showing/not showing a hidden event.

Maybe I'm missing something

Reply:
:see_no_evil: the param should be `false`.